config/routes.rb

```diff
@@ -3,12 +3,12 @@
   namespace :api do
     namespace :v1 do
       get '/plants/find', to: 'plant_search#index'
-      resources :users, only: [:index, :create, :update, :destroy] do
+      resources :users, only: [:index, :create, :show, :update, :destroy] do
```

Comment:
Ah okay I see now this was changed in this PR. I was under the impression that after we all discussed it, we were going to be using the route /user with the email being passed in params for the user dashboard path, instead of having user/:id